src/main/java/uk/gov/hmcts/darts/common/entity/CaseDocumentEntity.java

```diff
@@ -43,17 +48,37 @@ public class CaseDocumentEntity {
     @Column(name = "file_size", nullable = false)
     private Integer fileSize;
 
-    @Column(name = "checksum", nullable = false)
+    @Column(name = "checksum")
     private String checksum;
 
     @Column(name = "is_hidden", nullable = false)
     private boolean hidden;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "uploaded_by", nullable = false)
-    private UserAccountEntity uploadedBy;
+    @JoinColumn(name = "hidden_by")
+    private UserAccountEntity hiddenBy;
+
+    @Column(name = "hidden_ts")
+    private OffsetDateTime hiddenTs;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "created_by", nullable = false)
+    private UserAccountEntity createdBy;
+
+    @Column(name = "created_ts", nullable = false)
+    private OffsetDateTime createdTs;
+
+    @Column(name = "marked_for_manual_deletion", nullable = false)
+    private boolean markedForManualDeletion;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "marked_for_manual_del_by")
+    private UserAccountEntity markedForManualDelBy;
+
+    @Column(name = "marked_for_manual_del_ts")
+    private OffsetDateTime markedForManualDelTs;
 
-    @Column(name = "uploaded_ts", nullable = false)
-    private OffsetDateTime uploadedTs;
+    @Column(name = "retain_until_ts")
```

Comment:
these changes are defined in DMP-2658 - see my comments on there

Reply:
i mentioned this in the standup this morning and said that all the changes were done and just needed a review and that whoever was picking up 2658 didn't need to do them

Reply:
missed that - blaming it on my comms problem!